demo/src/app/components/+datepicker/demos/min-mode/min-mode.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-datepicker-min-mode',
+  templateUrl: './min-mode.component.html'
+})
+export class DemoDatepickerMinModeComponent {
+  bsValue: Date = new Date(2017, 7);
+
+  /* tslint:disable-next-line: no-any*/
+  bsConfig: any;
```

Comment:
could we add type for this config variable, likePartial<BsDaterangepickerConfig>? Or just move this to the template? For making component view more readable in the DEMO

Reply:
added